src/es5.js

```diff
@@ -15,7 +15,24 @@ if (isES5) {
         isES5: isES5,
         propertyIsWritable: function(obj, prop) {
             var descriptor = Object.getOwnPropertyDescriptor(obj, prop);
-            return !!(!descriptor || descriptor.writable || descriptor.set);
+
+            if (!descriptor) {
+                return true;
+            }
+
+            if (descriptor.set) {
```

Comment:
I think I misunderstood the purpose of a setter and distinction between having a setter and the purpose of the `writable` and `configurable` configs in this context.

It appears that `descriptor.set` presence is irrelevant for determining whether property is writable/ configurable.